SEUNAGBEYE/Math-Skill-Game
# #160830962 Setup webpack and babel

## How should this be manually tested?
- Fetch this branch with `git fetch origin ch-setup-webpack-and-babel-160830962:ch-setup-webpack-and-babel-160830962`
- Checkout to `ch-setup-webpack-and-babel-160830962`
- Run `yarn start`
- The app should be started, then visit the webpage and you should see `hello world`

### Any background context you want to provide?
NA


### What are the relevant pivotal tracker stories?
[#160830962](https://www.pivotaltracker.com/n/projects/2200608/stories/160831459)

### Screenshots (if appropriate)
NA
